src/plugins/mmapstorage/mmapstorage.c

```diff
@@ -0,0 +1,1323 @@
+/**
+ * @file
+ *
+ * @brief Source for mmapstorage plugin
+ *
+ * @copyright BSD License (see LICENSE.md or https://www.libelektra.org)
+ *
+ */
+#define _XOPEN_SOURCE 600
+
+/* -- Imports --------------------------------------------------------------------------------------------------------------------------- */
+
+#include "mmapstorage.h"
+#include "dynarray.h"
+
+#include <kdberrors.h>
+#include <kdbhelper.h>
+#include <kdblogger.h>
+#include <kdbmmap.h>
+#include <kdbprivate.h>
+
+#ifdef HAVE_KDBCONFIG_H
+#include "kdbconfig.h"
+#endif
+
+#include <assert.h>
```

Comment:
should not be needed (kdbassert.h instead)

Reply:
Done.